src/sentry/utils/migrations.py

```diff
@@ -0,0 +1,14 @@
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def clear_flag(Model, flag_name, flag_attr_name="flags"):
```

Comment:
It might be worth making `flag_name` a list, in case we want to reset mulitple at the same time

Reply:
@wedamija hmmm...I could do that but then the logic becomes a bit more complex. I'm not convinced it's worth the tradeoff (I don't expect flag clearing to happen _that_ often).